website/docs/guides/sl-snowflake-qs.md

```diff
@@ -0,0 +1,870 @@
+---
+title: "Quickstart for the dbt Cloud Semantic Layer and Snowflake"
+id: sl-snowflake-qs
+description: "Use this guide to build and define metrics, set up the dbt Cloud Semantic Layer, and query them using Google Sheets."
+sidebar_label: "Quickstart with the dbt Semantic Layer and Snowflake"
+meta:
+  api_name: dbt Semantic Layer APIs
+icon: 'guides'
+hide_table_of_contents: true
+tags: ['Semantic Layer', 'Snowflake', 'dbt Cloud','Quickstart']
+keywords: ['dbt Semantic Layer','Metrics','dbt Cloud', 'Snowflake', 'Google Sheets']
+level: 'Intermediate'
+recently_updated: true
+---
+
+<!-- The below snippets (or reusables) can be found in the following file locations in the docs code repository) -->
+import CreateModel from '/snippets/_sl-create-semanticmodel.md';
+import DefineMetrics from '/snippets/_sl-define-metrics.md';
+import ConfigMetric from '/snippets/_sl-configure-metricflow.md';
+import TestQuery from '/snippets/_sl-test-and-query-metrics.md';
+import ConnectQueryAPI from '/snippets/_sl-connect-and-query-api.md';
+import RunProdJob from '/snippets/_sl-run-prod-job.md';
+import SlSetUp from '/snippets/_new-sl-setup.md'; 
+
+<VersionBlock lastVersion="1.5">
+
+import DeprecationNotice from '/snippets/_sl-deprecation-notice.md';
+
+<DeprecationNotice />
+ 
+ </VersionBlock>
+
+## Introduction
+
+The [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl), powered by [MetricFlow](/docs/build/about-metricflow), simplifies the setup of key business metrics. It centralizes definitions, avoids duplicate code, and ensures easy access to metrics in downstream tools. MetricFlow helps manage company metrics easier, allowing you to define metrics in your dbt project and query them in dbt Cloud with [MetricFlow commands](/docs/build/metricflow-commands).
+
+
+import SLCourses from '/snippets/_sl-course.md';
+
+<SLCourses/>
+
+This quickstart guide is designed for dbt Cloud users using Snowflake as their data platform. It focuses on building and defining metrics, setting up the dbt Semantic Layer in a dbt Cloud project, and querying metrics in Google Sheets.
+
+If you're using a data platform other than Snowflake, this guide can also be applicable to you. You can adapt the setup for your specific platform by following the account setup and data loading instructions detailed on the following pages:
+
+- [BigQuery](https://docs.getdbt.com/guides/bigquery?step=2) (Steps 2 - 5)
+- [Databricks](https://docs.getdbt.com/guides/databricks?step=2) (Steps 2 - 4)
+- [Microsoft Fabric](https://docs.getdbt.com/guides/microsoft-fabric?step=2) (Steps 2 - 3)
+- [Redshift](https://docs.getdbt.com/guides/redshift?step=2) (Steps 2 - 4)
+- [Starburst Galaxy](https://docs.getdbt.com/guides/starburst-galaxy?step=2) (Steps 2 - 5)
+
```

Comment:
consider changing how this ^ list is constructed. maybe linkout to the steps/sections?

- the "(Steps x - y)" strings become a maintenance burden
- the reader has to flip back and forth between tabs/windows

Reply:
good point!

Reply:
it's prob likely that some tab back and forth might happen (maybe fetchin the (step 2-5) content so the user never leaves the page is a longer term experiment). but I've used tabs for this

![Screenshot 2024-04-19 at 12 36 50](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/9d710469-bb37-4563-a3c4-87c3afac18c8)